services/swagger/swagger.service.js

```diff
@@ -62,14 +66,22 @@ module.exports = class SwaggerValidatorService extends BaseJsonService {
     })
   }
 
+  transform(valMessages) {
+    if (
+      !valMessages ||
+      valMessages.length === 0 ||
+      valMessages.every(msg => msg.level === 'warning')
```

Comment:
Do we know what the full scope of potential `level` options are here? If it's something binary like, `warning` and `error` then we could probably use `.any(..)` instead. If we aren't sure what the potential options are, then this is probably the best approach 👍

Reply:
Taken from the [github site of the validator](https://github.com/swagger-api/validator-badge/blob/master/src/main/swagger/swagger.yaml)

 SchemaValidationError:
      type: object
      properties:
        level:
          type: string
        domain:
          type: string
        keyword:
          type: string
        message:
          type: string
        schema:
          $ref: '#/components/schemas/Schema'
        instance:
          $ref: '#/components/schemas/Instance'
        required:
          type: array
          items:
            type: string
        missing:
          type: array
          items:
            type: string